torch/_inductor/config.py

```diff
@@ -6,7 +6,11 @@
 
 
 def is_fbcode() -> bool:
-    return not hasattr(torch.version, "git_version")
+    try: 
+        from libfb.py import parutil # noqa
+        return not hasattr(torch.version, "git_version")
```

Comment:
Why 2nd check is needed, as `libfb.py` import will fail in all non Meta installations?
```suggestion
        return True
```

Reply:
libfb.py only available in fbcode, not avaialble in arvr.